Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- btrbk-0.31.3.orig/btrbk.conf.example
+++ btrbk-0.31.3/btrbk.conf.example
@@ -35,7 +35,7 @@ stream_buffer              256m
 # NOTE: btrbk does not autmatically create this directory, and the
 # snapshot creation will fail if it is not present.
 #
-snapshot_dir               _btrbk_snap
+snapshot_dir               _btrbk_snapshots
 
 # Always create snapshots. Set this to "ondemand" to only create
 # snapshots if the target volume is reachable. Set this to "no" if
@@ -137,46 +137,46 @@ archive_preserve        12m 10y
 # be mounted first:
 #  $ mount -o subvolid=5 /dev/sda1 /mnt/btr_pool
 #
-volume /mnt/btr_pool
+#volume /mnt/btr_pool
   # No action if external disk is not attached
-  snapshot_create ondemand
+#  snapshot_create ondemand
 
   # Target for all subvolume sections:
-  target /mnt/btr_backup/_btrbk
+#  target /mnt/btr_backup/_btrbk
 
   # Some default btrfs installations (e.g. Ubuntu) use "@" for rootfs
   # (mounted at "/") and "@home" (mounted at "/home"). Note that this
   # is only a naming convention.
-  #subvolume @
-  subvolume root
-  subvolume home
-  subvolume kvm
+#  subvolume @
+#  subvolume root
+#  subvolume home
+#  subvolume kvm
     # Use different retention policy for kvm backups:
-    target_preserve 7d 4w
+#    target_preserve 7d 4w
 
 
 # Backup data to external disk as well as remote host
-volume /mnt/btr_data
-  subvolume  data
+#volume /mnt/btr_data
+#  subvolume  data
     # Always create snapshot, even if targets are unreachable
-    snapshot_create always
-    target /mnt/btr_backup/_btrbk
-    target ssh://backup.my-remote-host.com/mnt/btr_backup
+#    snapshot_create always
+#    target /mnt/btr_backup/_btrbk
+#    target ssh://backup.my-remote-host.com/mnt/btr_backup
 
 
 # Backup from remote host, with different naming
-volume ssh://my-remote-host.com/mnt/btr_pool
-  subvolume data_0
-    snapshot_dir   snapshots/btrbk
-    snapshot_name  data_main
-    target /mnt/btr_backup/_btrbk/my-remote-host.com
+#volume ssh://my-remote-host.com/mnt/btr_pool
+#  subvolume data_0
+#    snapshot_dir   snapshots/btrbk
+#    snapshot_name  data_main
+#    target /mnt/btr_backup/_btrbk/my-remote-host.com
 
 
 # Resume backups from remote host which runs its own btrbk instance
 # creating snapshots for "home" in "/mnt/btr_pool/btrbk_snapshots".
-volume ssh://my-remote-host.com/mnt/btr_pool
-  snapshot_dir           btrbk_snapshots
-  snapshot_create        no
-  snapshot_preserve_min  all
-  subvolume home
-    target /mnt/btr_backup/_btrbk/my-remote-host.com
+#volume ssh://my-remote-host.com/mnt/btr_pool
+#  snapshot_dir           btrbk_snapshots
+#  snapshot_create        no
+#  snapshot_preserve_min  all
+#  subvolume home
+#    target /mnt/btr_backup/_btrbk/my-remote-host.com
